libafl_bolts/Cargo.toml

```diff
@@ -165,6 +177,12 @@ serial_test = { workspace = true, optional = true, default-features = false, fea
 # Document all features of this crate (for `cargo doc`)
 document-features = { workspace = true, optional = true }
 
+arbitrary-int = { version = "1.2.7", optional = true }
+bitbybit = { version = "1.3.2", optional = true }
```

Comment:
Do we need all of these new deps?

Reply:
I use specifically these two for C bitfields, and I like this one a lot because of its API:
https://github.com/AFLplusplus/LibAFL/blob/59ae76090a4ccf299e5b856d7ec12fd8a1924296/libafl_bolts/src/intel_pt.rs#L706-L717

(It is a shame that Rust doesn't have bitfields in `core` and there isn't even a de-facto standard crate 🫤)

Reply:
The follwing ones yeah we need those, `raw-cpuid` maybe is not 100% essential but helps

Reply:
Just double-checking. Up to you, in the end.
For bitfields we already use a crate here:
https://github.com/AFLplusplus/LibAFL/blob/d1c746a0a2d697e7ab6294dca8b9fbc908e92251/libafl/Cargo.toml#L218

Reply:
Yes thanks, I saw that one but I decided against it after trying to use it, if I remember correctly

Reply:
it would be nice to unify usage of bitfields if it's doable.
it's the kind of thing that quickly becomes confusing if we have multiple ways to do the same things

Reply:
yes, if you prefer the other trait for one reason or another, change both

Reply:
Also describe why you prefer it